README.md

```diff
@@ -2,179 +2,128 @@
-sudo node /usr/lib/node_modules/gtp2ogs/gtp2ogs.js --beta --apikey 5rg46ze84f68e4g6v4e4g68es4g8 --username testbot --debug --startupbuffer 2 --bans 454,someplayer,489,512 --noclock --unrankedonly --maxconnectedgames 10 --maxconnectedgamesperuser 1 --minrank 5d --noautohandicapranked --maxhandicap 0 --fakerank 8d --boardsizes 19 --komis 7.5 --speeds live --maxmaintimelive 600 --maxperiodsliveranked 5 --maxperiodsliveunranked 10 --maxperiodtimeliveranked 60 --maxperiodtimeliveunranked 180 -- /home/amd2019/PhoenixGo/bazel-bin/mcts/mcts_main --gtp --config_path=/home/amd2019/PhoenixGo/etc/mcts_1gpu_beta.conf --v=1
-```
+For example:
 
-### Submit Move Button
+`
+node ~/gtp2ogs-node/node_modules/gtp2ogs/gtp2ogs.js --beta --apikey 5rg46ze84f68e4g6v4e4g68es4g8 --username somebot --debug --startupbuffer 2 --bans 454,someplayer,489,512 --noclock --unrankedonly --maxconnectedgames 10 --maxconnectedgamesperuser 1 --minrank 5d --noautohandicapranked --maxhandicap 0 --boardsizes 19 --komis 7.5 --speeds live --maxmaintimelive 600 --maxperiodsliveranked 5 --maxperiodsliveunranked 10 --maxperiodtimeliveranked 60 --maxperiodtimeliveunranked 180 -- /home/amd2020/PhoenixGo/bazel-bin/mcts/mcts_main --gtp --config_path=/home/amd2020/PhoenixGo/etc/mcts_1gpu_beta.conf --logtostderr --v=1
+`
 
-To avoid accidental misclicks while spectating a game from
-your bot account,
+## Submit Move Button
 
-see: [Submit-Move Button](https://github.com/wonderingabout/gtp2ogs-tutorial#important-submit-move-button)
+To avoid accidental misclicks while spectating a game from your bot account, see
+ [Submit-Move Button](https://github.com/wonderingabout/gtp2ogs-tutorial#important-submit-move-button).
 
-### Show winrate and variations ingame
+## Show winrate and variations ingame
 
 **gtp2ogs has default support for showing ingame winrate and variations for some AI.**
 
 see: [--ogspv](/docs/OPTIONS-LIST.md/#ogspv)
 
 ## Community Involvment
 
-- You may be interested in [Contributing](/docs/CONTRIBUTING.md)
 - You may also use [Issues](https://github.com/online-go/gtp2ogs/issues)
-to report issues.
-
-- You may be interested in joining the Discord chat:
-In leela zero discord (`#bots` channel): <https://discord.gg/HZ23Cp9>
+ to report issues.
+- Contributing in most welcome, if you want to add features or submit fixes
+ we'd be glad to review them with you and make gtp2ogs more awesome!
+- You may also be interested in joining the Discord chat:
+ in leela zero discord (`#bots` channel): <https://discord.gg/HZ23Cp9>
 
 This discord can also be useful if you want to have fast and quick,
-interactive chat, or ask simple and quick questions (github Issues are
-preferred for long problems because they live a track that can be useful
-later)!
+ interactive chat, or ask simple and quick questions, however github issues
+ are preferred for long problems because they live a track that can be useful
+ later)
+
+## Developer Notes
+
+You can find notes for developers [in DEV.md](/docs/DEV.md).
```

Comment:
[here](/docs/DEV.md)

Reply:
i dont understand, the link seems to correctly redirect to `/docs/DEV.md` on my local branch

Reply:
I mean, link with the word 'here'

Reply:
ah i see, fixed :+1: